tox.ini

```diff
@@ -52,7 +52,7 @@ deps =
     pytest
     pytest-cov
 commands =
-    pytest --cov={envsitepackagesdir}/junifer --cov-report=xml --cov-report=term
```

Comment:
We either test and evaluate coverage on `{envsitepackagesdir}` or on the _checked out_ code.

My take is that this way we are testing what we ship in the installable package.

Reply:
Yeah it stopped reporting coverage out of the blue so as long as we are back, we are good.